exercises/5_OOP-and-Intents/oop/src/Clarinet.java

```diff
@@ -0,0 +1,10 @@
+/**
+ * Created by charlynbuchanan on 4/30/15.
+ */
+public class Clarinet extends ReedInstrument{
+    @Override
+    public String play()
+    {
+        return null;
```

Comment:
Clean. Can also return a string along the lines of "Instrument plays" For testing purposes.